security.md

```diff
@@ -93,6 +94,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2025-9572"></a>CVE-2025-9572: Information disclosure via GraphQL resolver taxonomy scoping bypass
+
+A vulnerability was found in Foreman where the GraphQL resolver implementation does not apply taxonomy scoping filters.
+This allows authenticated users to view location and organizational structure data beyond their authorized scope.
+
+* Affects Foreman 1.22.0 and higher
+* Fix released in Foreman 3.16.2, 3.17.0, 3.18.0
```

Comment:
Do we need to explicitly mention 3.18.0 if it went out (or will go out shortly) in 3.17.0?